app/models/user_log.rb

```diff
@@ -0,0 +1,6 @@
+class UserLog < ApplicationRecord
```

Comment:
Usually, for authentication, we use `User` or `Account`.  Please choose one of these and rebuild this code.